example-notebooks/binary-classifier/classifier_bin.ipynb

```diff
@@ -192,7 +192,7 @@
     "<li>the filepath to predictions definitions, that specify cohorts, scores, and features to consider.</li>\n",
     "</ol>\n",
     "\n",
-    " have created: \n",
+    "We have created: \n",
```

Comment:
This should already be fixed see https://github.com/epic-open-source/seismometer/blob/main/example-notebooks/binary-classifier/classifier_bin.ipynb

Reply:
Confirmed